doc/releases/changelog-0.35.0.md

```diff
@@ -70,6 +195,9 @@
     >>> X(0)
     X(0)
     ```
+    
+    The original long-form names `Identity`, `PauliX`, `PauliY`, and `PauliZ` remain available, but
+    use of the short-form names is now recommended.
 
   * PennyLane will try to automatically work with a Pauli representation of operators when
     available. The Pauli representation can be optionally accessed via `op.pauli_rep`:
```

Comment:
Same here; the lead should be the advantage rather than the implementation detail. That is, using the new operator arithmetic, sums/linear combinations of Pauli operators will lead to a more efficient representation, and more efficient arithmetic.

in fact, I don't think we even need to focus on the implementation detail here (e.g., showing that the type is a PauliSentence)

Reply:
Also commutator is a very cool feature, and should be higher!